apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -6,14 +6,14 @@ import {updatePointerBlockImage} from '@cdo/apps/blockly/addons/cdoSpritePointer
 import CdoFieldFlyout from '@cdo/apps/blockly/addons/cdoFieldFlyout';
 import {spriteLabPointers} from '@cdo/apps/p5lab/spritelab/blockly/constants';
 import {blocks as behaviorBlocks} from './behaviorBlocks';
+import {BLOCK_TYPES, NO_OPTIONS_MESSAGE} from '@cdo/apps/blockly/constants';
+import {FALSEY_DEFAULT, readBooleanAttribute} from '@cdo/apps/blockly/utils';
+import {editButtonHandler} from './proceduresBlocks';
 
 const INPUTS = {
```

Comment:
This constant was nested in between two sets of imports, so I moved it to the bottom/all the imports above it.